src/inc/quic_driver_connection.h

```diff
@@ -0,0 +1,376 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+--*/
+
+#pragma once
+
+#define QUIC_TEST_APIS 1
+#include "quic_platform.h"
+#include "quic_trace.h"
+
+#ifdef _WIN32
```

Comment:
Instead of this, let's make the caller only include the file in _WIN32. Then we can remove this here.

Reply:
That massively breaks the BVT tests. It depends on the empty implementations being there.